src/test/java/edu/cornell/kfs/sys/service/impl/AwsSecretServiceImplIntegrationTest.java

```diff
@@ -78,24 +83,59 @@ void testBooleanSetAndGet() {
     }
     
     @Test
-    void testPojo() throws JsonMappingException, JsonProcessingException {
+    void testBooleanSetAndGetWithOutCache() {
+        boolean initialValue = awsSecretServiceImpl.getSingleBooleanFromAwsSecret(SINGLE_BOOLEAN_SECRET_KEY_NAME, false);
+        boolean expectedNewBoolean = !initialValue;
+        awsSecretServiceImpl.updateSecretBoolean(SINGLE_BOOLEAN_SECRET_KEY_NAME, false, expectedNewBoolean);
+        awsSecretServiceImpl.clearCache();
+        
+        boolean actualNewBoolean = awsSecretServiceImpl.getSingleBooleanFromAwsSecret(SINGLE_BOOLEAN_SECRET_KEY_NAME, false);
+        
+        assertEquals(expectedNewBoolean, actualNewBoolean);
+    }
+    
+    @Test
+    void testPojoWithCache() throws JsonMappingException, JsonProcessingException {
         String newUniqueString = UUID.randomUUID().toString();
-        Date newDate = new Date(Calendar.getInstance().getTimeInMillis());
+        Date newDate = new Date(Calendar.getInstance(Locale.US).getTimeInMillis());
         
         AwsSecretPojo pojo = awsSecretServiceImpl.getPojoFromAwsSecret(BASIC_POJO_SECRET_KEY_NAME, false, AwsSecretPojo.class);
-        LOG.info("testPojo, pojo: " + pojo);
+        LOG.info("testPojoWithCache, pojo: " + pojo);
+        pojo.setChangeable_string(newUniqueString);
+        pojo.setUpdate_date(newDate);
+        boolean newBooleanTest = !pojo.isBoolean_test();
+        pojo.setBoolean_test(newBooleanTest);
+        awsSecretServiceImpl.updatePojo(BASIC_POJO_SECRET_KEY_NAME, false, pojo);
+        
+        AwsSecretPojo pojoNew = awsSecretServiceImpl.getPojoFromAwsSecret(BASIC_POJO_SECRET_KEY_NAME, false, AwsSecretPojo.class);
+        assertEquals(newUniqueString, pojoNew.getChangeable_string());
+        assertEquals(BASIC_POJO_STATIC_STRING_VALUE, pojoNew.getStatic_string());
+        assertEquals(BASIC_POJO_NUMBER_VALUE, pojoNew.getNumber_test());
+        assertEquals(pojo.getUpdate_date(), pojoNew.getUpdate_date());
+        assertEquals(pojo.isBoolean_test(), pojoNew.isBoolean_test());
+    }
+    
+    @Test
+    void testPojoWithOutCache() throws JsonMappingException, JsonProcessingException {
+        String newUniqueString = UUID.randomUUID().toString();
+        Date newDate = new Date(Calendar.getInstance().getTimeInMillis());
```

Comment:
This line should use the Locale.US setup for the Calendar instance, like in the method above this one.

Reply:
OK